src/lib/expert_partitioner/dialogs/format.rb

```diff
@@ -61,27 +57,21 @@ def run
       end
     end
 
-
-  private
+    private
```

Comment:
to be honest I actually very dislike this indentation, but if storage subteam agreed on it, no problem.
For me main reason why I do not like is that it is global modificator and when inlined into methods indentation it can be overlooked so

```
class C
  trash
  ....
  ...

  private
  
  ....
  trash
  ...
end
```
is easier to overlook what is private then
```
class C
  trash
  ....
  ...

private
  
  ....
  trash
  ...
end
```

so my project enforce style with outer indentation of private